vendor/licenses/git_submodule/Portugol.dep.yml

```diff
@@ -0,0 +1,138 @@
+---
+name: Portugol
+version: 2.0
+type: git_submodule
+homepage: https://github.com/UNIVALI-LITE/Portugol-Studio
+license: LGPL-3.0
```

Comment:
This license file needs to be the license file of the grammar. See my first comment.